File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
a = int(input())
b = list(map(int, input().split()))
max = b[0]
min = b[0]
for i in range(1,a):
    if max < b[i]:
        max = b[i]
    if min > b[i]:
        min = b[i]
print("%d %d"%(min, max))

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <stdio.h>

int main(int argc, const char * argv[]) {
    int t, i, min, max;
    scanf("%d", &t);
    int arr[t];
    for(i=0; i<t; i++){
        scanf("%d", &arr[i]);
    }
    min = arr[0];
    max = arr[0];
    for(i=0;i<t;i++){
        if(min>arr[i]){
            min = arr[i];
        }
        if(max<arr[i]){
            max = arr[i];
        }
    }
    printf("%d %d", min, max);
    return 0;
}